src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -5,8 +5,36 @@ ifdef::cheri_standalone_spec[]
 WARNING: This chapter will appear in the unpriv spec after the RV32I chapter.
 endif::[]
 
-This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I (or RV32E/RV64E) integer instruction sets with CHERI.
-If the hart supports RV32E/RV64E instead of RV32I/RV64I, {cheri_base_ext_name} the restrictions from <<rv32e>> apply and 16 general purpose registers instead of 32 -- but all other {cheri_base_ext_name} changes are identical so the remainder of this chapter only refers to extending RV32I/RV64I behavior.
+This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I instruction sets with CHERI.
+Other standard options to bases are also supported such as E (16-registers), Zfinx and endianess.
+
+=== CHERI encoding formats
+
+CHERI implementations are required to choose a capability encoding format.
+The CHERI base ISA name is extended with the capability encoding format in use, and each implementation only supports one.
+
+The definition of all CHERI base ISAs include sufficient details of a capability encoding format to give enough details to allow execution of all base ISA instructions.
```

Comment:
That was the intent of my parametric base proposal, but that didn't seem hugely popular.  Can you clarify what you mean by "enough details"?

Reply:
It needs enough details to cover loads, stores, branches, capability ADD etc. so needs to have enough permissions and bounds details included to cover all of those.

Reply:
That's the answer to the first question I expected, so I think that's good.  How about the 2nd?

Reply:
I think having meaningful parameters in a CSR would definitely be good. `ycfg` was just a minimal proposal, doing something more structured sounds good to me.

Reply:
I'm very much in favour of fast track later - if it's not critically important now......